setupext.py

```diff
@@ -922,6 +922,13 @@ def add_flags(self, ext, add_sources=True):
                     os.path.join('extern', 'agg24-svn', 'src', x) for x in agg_sources)
 
 
+# For FreeType2 and libpng, we add a separate checkdep_foo.c source to at the
+# top of the extension sources.  This file is compiled first and immediately
```

Comment:
Can you actually guarantee this if run with `-j`?

Reply:
I believe so: looking at https://github.com/python/cpython/blob/master/Lib/distutils/command/build_ext.py#L450 indicates that parallel building means that multiple extensions are built in parallel, but each individual extension is still built serially.  (Right?)